apps/ui/src/networks/evm/constants.ts

```diff
@@ -16,7 +16,7 @@ import IHUserCircle from '~icons/heroicons-outline/user-circle';
 import IHLightningBolt from '~icons/heroicons-outline/lightning-bolt';
 import { MAX_SYMBOL_LENGTH } from '@/helpers/constants';
 
-export const MAX_MERKLE_VOTING_STRATEGIES = 10;
+export const MAX_MERKLE_VOTING_STRATEGIES = 10; // !NOTE: Update this for mainnet
```

Comment:
@rajkharvar can push this change in this PR? we can keep it at 100 as discussed

Reply:
Done in [93159a1](https://github.com/razor-network/sx-monorepo/pull/9/commits/93159a1b7ad416776e94ecbdcc6720889236bdd5)